### PR TITLE
Upgrade mjolnir (0.1.17 -> 0.1.18) and implement self building 

### PR DESCRIPTION
**Pull request by aaronraimist**

### CODE REVIEW

File: roles/matrix-bridge-mx-puppet-twitter/tasks/init.yml [hunk 1]
```diff
@@ -68,10 +68,3 @@
       URL endpoint to the matrix-mx-puppet-twitter container.
       You can expose the container's port using the `matrix_mx_puppet_twitter_container_http_host_bind_port` variable.
   when: "matrix_mx_puppet_twitter_enabled|bool and matrix_nginx_proxy_enabled is not defined"
-
```
Comment (spantaleev) [inline roles/matrix-bridge-mx-puppet-twitter/tasks/init.yml:71]: Why does this PR remove these tasks from other roles?

Comment (aaronraimist) [reply roles/matrix-bridge-mx-puppet-twitter/tasks/init.yml:71]: They are redundant. There are two duplicate checks in each of those files.

Comment (spantaleev) [reply roles/matrix-bridge-mx-puppet-twitter/tasks/init.yml:71]: Oh, interesting! :laughing:

**Comment by spantaleev** [comment]
Thank you! :+1:

**Pull request merged**